src/components/html/EcApplication.html

```diff
@@ -1,41 +1,84 @@
-            title="Tem certeza que deseja excluir o aplicativo?"
-            @confirm="deleteApp"
-            okText="Sim"
-            cancelText="Não"
-            v-if="isInstalled"
+<div class="ec-application">
+  <a-row :gutter="24">
+    <div v-if="loading">
+      <a-col :md="24" :lg="7">
+        <a-card :bordered="false">
+          <div class="text-center">
+            <div>
+              <img :src="icon" style="margin: 0 auto;width: 104px;height: 104px;margin-bottom: 20px;border-radius: 50%;overflow: hidden;">
+            </div>
+            <div style="margin-bottom: 5px">{{ title }}</div>
+            <div>{{ shortDescription }}</div>
+          </div>
+
+          <div style="text-align: center; margin-bottom: 5px" class="mt-4">
+            <span>
+              <a-button type="primary" shape="round" @click="installApp" v-if="!isInstalled">
+                <a-icon type="download" />{{ i19install }}</a-button>
+              <a-button type="danger" shape="round" @click="uninstallApp" v-else>
+                <a-icon type="delete" />{{ i19uninstall }}</a-button>
+            </span>
+          </div>
+
+          <div style="text-align: center; margin-bottom: 5px">
+            <p>{{ price }}</p>
+          </div>
+
+          <div>
+            <p class="d-flex align-items-center">
+              <a-icon type="user" style="margin-right: 1rem;"/>
+              {{ i19author }} : {{ author }}
+            </p>
+            <p class="d-flex align-items-center">
+              <a-icon type="interation" style="margin-right: 1rem;"/>
+              {{ i19version }} : {{ version }}
+            </p>
+            <p class="d-flex align-items-center">
+              <a-icon type="global" style="margin-right: 1rem;"/>
+              <a :href="website">Website</a>
+            </p>
+          </div>
+        </a-card>
+      </a-col>
+
+      <a-col :md="24" :lg="17">
+        <a-card 
+          style="width:100%" 
+          :bordered="false" 
+          :tabList="tabListNoTitle" 
+          :activeTabKey="activeTabKey"
+          @tabChange="key => handleTabChange(key, 'activeTabKey')"
+        >
+
+          <a-card 
+            :bordered="false" 
+            v-if="activeTabKey === 'description'"
+            key="app-description"
           >
-            <a-icon type="delete" title="Excluir aplicativo" class="ec-application__actions"/>
-          </a-popconfirm>
-        </template>
-        <a-card-meta :title="title" :description="shortDescription">
-          <a-avatar
-            slot="avatar"
-            :src="icon"
-          />
-        </a-card-meta>
-      </a-card>
+            <vue-markdown>{{description}}</vue-markdown>
+          </a-card>
+
+          <article v-else-if="activeTabKey === 'settings'" key="app-settings"></article>
+
+          <article v-else-if="activeTabKey === 'related'" key="app-related">
+            <div class="row">
+              <div 
+                class="col-6 col-md-4 col-lg-4" 
```

Comment:
Neste caso o `col-lg-4` não muda nada...

```suggestion
                class="col-6 col-md-4" 
```